proto/v1/database_role.proto

```diff
@@ -45,6 +45,11 @@ message DatabaseRole {
   DatabaseRoleAttribute attribute = 5;
 }
 
+// DatabaseRoleList is the API message for role list.
+message DatabaseRoleList {
+  repeated DatabaseRole role_list = 1;
```

Comment:
I have no idea how to marshal a slice through the protojson (`protojson.Marshal(roleList)` will throw `does not implement protoreflect.ProtoMessage` error)

Reply:
We need to use ListRoleResponse instead with a list of roles.

Reply:
Yeap, so I add this `DatabaseRoleList` to wrap the list